components/server/src/server.ts

```diff
@@ -268,6 +270,9 @@ export class Server {
         // Start periodic jobs
         this.disposables.push(this.jobRunner.start());
 
+        // Start workspace start controller
+        this.disposables.push(await this.workspaceStarter.start());
```

Comment:
It means this lob is running in each server? Do we want to do it as a job instead to have less concurrency?

Reply:
We could! I understood "simpler" yesterday, so started out with this :wink:

Reply:
I believe using a job is actually simpler, because it doesn'T introdcue another way of running ongoing workloads.